keps/sig-node/2902-cpumanager-distribute-cpus-policy-option/README.md

```diff
@@ -182,12 +209,33 @@ No changes. Existing container will not see their allocation changed. New contai
 
 - A specific e2e test will demonstrate that the default behaviour is preserved when the feature gate is disabled, or when the feature is not used (2 separate tests)
 
+### Rollout, Upgrade and Rollback Planning
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+- A rollout or rollback can fail if the feature gate and the policy option are not configured properly and kubelet fails to start.
```

Comment:
How can one misconfigure feature gate and policy option?

Reply:
It could be due to human error, such as a typo in the policy option or the feature gate. Additionally, there might be a misalignment between the feature gate that controls the option. For example, if `CPUManagerPolicyBetaOptions` is set to false, it could unintentionally prevent access to the desired policy.